wsmaster/integration-tests/mysql-tck/pom.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2012-2018 Red Hat, Inc.
+    This program and the accompanying materials are made
+    available under the terms of the Eclipse Public License 2.0
+    which is available at https://www.eclipse.org/legal/epl-2.0/
+
+    SPDX-License-Identifier: EPL-2.0
+
+    Contributors:
+      Red Hat, Inc. - initial API and implementation
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>integration-tests-parent</artifactId>
+        <groupId>org.eclipse.che.core</groupId>
+        <version>6.12.0-SNAPSHOT</version>
+    </parent>
+    <groupId>org.eclipse.che.core</groupId>
+    <artifactId>mysql</artifactId>
+    <packaging>pom</packaging>
+    <name>Integration Tests :: MySQL</name>
+    <modules>
+        <module>mysql-tck</module>
+        <module>mysql-support</module>
+    </modules>
+    <properties>
+        <db.image.name>centos/mysql-57-centos7</db.image.name>
+        <jdbc.admin.name>root</jdbc.admin.name>
+        <jdbc.admin.password>che2</jdbc.admin.password>
+        <jdbc.driver>com.mysql.cj.jdbc.Driver</jdbc.driver>
+        <jdbc.url>jdbc:mysql://${docker.host.address}:${jdbc.port}/che</jdbc.url>
+        <jdbc.user.name>user</jdbc.user.name>
+        <jdbc.user.password>che1</jdbc.user.password>
+    </properties>
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>mysql</groupId>
+                <artifactId>mysql-connector-java</artifactId>
+                <version>8.0.12</version>
```

Comment:
That needs to be CQ-ed and moved into che-parent (we can do that btw)

Reply:
https://dev.eclipse.org/ipzilla/show_bug.cgi?id=17680